DigitalLearningSolutions.Data/Services/FrameworkNotificationService.cs

```diff
@@ -198,34 +197,6 @@ public void SendReviewOutcomeNotification(int reviewId)
             emailService.SendEmail(new Email(emailSubjectLine, builder, outcomeNotification.OwnerEmail, outcomeNotification.UserEmail));
         }
 
-        public void SendSupervisorDelegateInvite(int supervisorDelegateId, int adminId)
-        {
-            var supervisorDelegate = supervisorService.GetSupervisorDelegateDetailsById(supervisorDelegateId, adminId, 0);
-            string emailSubjectLine = "Invite from Supervisor - Digital Learning Solutions";
-            var builder = new BodyBuilder();
-            var dlsUrlBuilder = GetDLSUriBuilder();
-            if (supervisorDelegate.CandidateID == null)
```

Comment:
We need to keep this notification when adding an email to the staff list that doesn't match a candidate record at the centre. The wording of this email doesn't need changing in this scenario.